src/styles/extension-styles.tsx

```diff
@@ -0,0 +1,35 @@
+import { css } from '@emotion/react';
+
+export const extensionStyles = css`
```

Comment:
Would reconsider the name. The whole thing is the "extension". These styles relate to the popup right? Or, no? I see there's another popup one. So I'm not really sure what this is for. If they're global, I wouldn't nest under a `.mode__extension` class.

(Side note unrelated to this PR: the `<PopupContainer />` is a really bad component name, as it's also used in the full page mode)

Reply:
Are we using any naming convention throughout the code base? something like BEM for example

Reply:
I agree, this naming convention was already here but happy to change it. I was confused at first too, but `popup` is the center modal and `extension` is the popup from the browser. Look in the `public > html` files. I'll check to see if popup is even being used. I think maybe people don't understand that `PopupContainer` is shared b/w the full-page and extension popup, the only thing that changes is the class naming in the global styles. It is a bit confusing but it is how it was setup initially.

Reply:
Nvm, I reread your comment up `PopupContainer` ...I see you get it is shared b/w the modes.

Reply:
> Are we using any naming convention throughout the codebase? something like BEM for example

What does `BEM` stand for?

Reply:
[Block Element Modifier](http://getbem.com/), the `__`s and `--`s. We're kind of following it here, but generally we're not writing CSS, @andresgalante, so we're not enforcing any particular convention.

Reply:
yeap, I am aware that since we are doing CSS in JS naming becomes irrelevant. Naming is impossible, good luck with this one :)

Reply:
Looked into this ...we need all 3 modes to capture diff styling for the full-page, popup modal (center), and the browser extension dropdown. I am open to suggestions on renaming the extension mode but the popup mode is different. They have to be nested under each mode class bc the styling changes for each one. Also, any suggestions on a new name for `PopupContainer`? I don't think anything nested in the `components > popup` really makes any sense.

Reply:
I added an image of the popup modal to the description.

Reply:
@kyranjamie I changed to `AppContainer`, what do you think? It might be confusing bc it is used in so many places? Can you look this over again now with the name changes? Hoping to get it merged into the feature branch today if can.